src/bin/hue_get_all_lights.rs

```diff
@@ -8,7 +8,7 @@ fn main() {
         println!("usage : {:?} <username>", args[0]);
         return;
     }
-    let bridge = ::hueclient::bridge::Bridge::discover_required().with_user(args[1].to_string());
+    let bridge = ::hueclient::Bridge::discover_required().with_user(args[1].to_string());
```

Comment:
Please remove the pre-2018 prefix when you have the opportunity to without introducing more changes.